Content.Shared/_Goobstation/Clothing/Components/ClothingGrantComponentComponent.cs

```diff
@@ -0,0 +1,15 @@
+using Robust.Shared.Prototypes;
+
+namespace Content.Shared._Goobstation.Clothing.Components
+{
+    [RegisterComponent]
+    public sealed partial class ClothingGrantComponentComponent : Component
+    {
+        [DataField("component", required: true)]
```

Comment:
Named fields bad

Reply:
but this is the rare case where the name is different from the variable name, lmk if changing this would break the goob port too hard